tests/SpiffWorkflow/camunda/data/training_workflow.bpmn

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
just wanted to make sure you intended to include training workflow as a part of this change.

Reply:
No, that was a mistake.  Removed.